README.md

```diff
@@ -42,9 +42,11 @@ As a starting point for your own project take a look at https://github.com/ether
 
 ##### Running from command line:
 ```
-> git clone https://github.com/ethereum/ethereumj
-> cd ethereumj
-> ./gradlew run [-PmainClass=<sample class>]
+git clone https://github.com/ethereum/ethereumj
+cd ethereumj
+vim ethereumj-core/src/main/resources/ethereumj.conf # adjust ethereumj.conf to your needs
```

Comment:
`resources/ethereumj.conf` is intended for default config values and is not subject to be changed. 
There is a plenty of ways to supply the user config including `user.conf` as a resource